plugins/modules/docker_secret.py

```diff
@@ -246,39 +293,47 @@ def create_secret(self):
 
         return secret_id
 
+    def remove_secret(self, secret):
+        try:
+            if not self.check_mode:
+                self.client.remove_secret(secret['ID'])
+        except APIError as exc:
+            self.client.fail("Error removing secret %s: %s" % (self.name, to_native(exc)))
```

Comment:
Shouldn't it be:
```suggestion
            self.client.fail("Error removing secret %s: %s" % (secret['Spec']['Name'], to_native(exc)))
```
?

Reply:
Fixed in [8648e89](https://github.com/ansible-collections/community.docker/pull/293/commits/8648e89).